stubs/PyYAML/yaml/nodes.pyi

```diff
@@ -1,31 +1,37 @@
-from typing import Any
+from typing import Any, ClassVar
+
+from yaml.error import Mark as Mark
 
 class Node:
-    tag: Any
+    tag: str
     value: Any
-    start_mark: Any
-    end_mark: Any
-    def __init__(self, tag, value, start_mark, end_mark) -> None: ...
+    start_mark: Mark | Any
+    end_mark: Mark | Any
+    def __init__(self, tag: str, value: Any, start_mark: Mark | None, end_mark: Mark | None) -> None: ...
 
 class ScalarNode(Node):
-    id: Any
-    tag: Any
+    id: ClassVar[str]
+    tag: str
     value: Any
-    start_mark: Any
-    end_mark: Any
-    style: Any
-    def __init__(self, tag, value, start_mark=..., end_mark=..., style=...) -> None: ...
+    start_mark: Mark | Any
+    end_mark: Mark | Any
```

Comment:
Nit: Since they are already defined on the base class, they don't need to be defined here. Same for the classes below.